### PR TITLE
silence release prompter & confirm release script works locally

### PR DESCRIPTION
**Pull request by outkaj**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -129,11 +130,20 @@ lazy val openlawCore = crossProject(JSPlatform, JVMPlatform)
   .settings(releaseSettings: _*)
   .enablePlugins(WartRemover)
 
+
+lazy val version = "sbt version"!
+
+addCommandAlias("releaseCore", ";project openlawCore ;release release-version ${version} next-version ${version-SNAPSHOT} with-defaults")
```
Comment (mroth) [inline build.sbt]: Cool!

File: scripts/release.sh [hunk 2]
```diff
@@ -31,7 +31,11 @@ fi
 # Publish to Bintray. Will do nothing by default for safety unless overridden
 # via RELEASE_TRIGGER=true.
 if [ "$RELEASE_TRIGGER" = "true" ]; then
-    sbt release
+    # release Scala & then ScalaJS to avoid issue where release-with-defaults only releases Scala lib
+    # the next-version flag is to silence SBT's interactive release shell prompt - it doesn't actually alter the version
```
Comment (mroth) [inline scripts/release.sh]: This comment left over from previous version? (I don't see this flag here anymore)

File: build.sbt [hunk 3]
```diff
@@ -129,11 +130,20 @@ lazy val openlawCore = crossProject(JSPlatform, JVMPlatform)
...
+addCommandAlias("releaseCoreJS", ";project openlawCoreJS ;release release-version ${version} next-version ${version-SNAPSHOT} with-defaults")
```
Comment (mroth) [inline build.sbt]: Mostly curious, but can these recurse? Like, could we then add a third alias that did `addCommandAlias("releaseBoth", "releaseCore releaseCoreJs")`? (Maybe not worth it unless it specifically let us override `'release'` more just curious for future usage.

File: build.sbt [hunk 4]
```diff
@@ -129,11 +130,20 @@ lazy val openlawCore = crossProject(JSPlatform, JVMPlatform)
   .settings(releaseSettings: _*)
   .enablePlugins(WartRemover)
 
+
+lazy val version = "sbt version"!
```
Comment (mroth) [inline build.sbt]: I'm wondering if there is any way we can avoid shelling out to get this info (which can break in unexpected ways due to $PATH issues, etc). Does SBT not have access to this info in a local variable already?

Comment (mroth) [reply build.sbt]: Looking at the sbt-git source code, I think we _may_ be able to get this via `git.gitDescribedVersion`? https://github.com/sbt/sbt-git/blob/master/src/main/scala/com/typesafe/sbt/SbtGit.scala#L120

File: scripts/release.sh (hunk 2 above)
Comment (outkaj) [reply scripts/release.sh]: yes, left over! I'll move to `build.sbt` where it should be

File: build.sbt (hunk 3 above)
Comment (outkaj) [reply build.sbt]: yes I think so. I like that idea, i'll try it out

File: build.sbt (hunk 1 above)
Comment (outkaj) [reply build.sbt]: yeah i didn't know sbt could do that!

File: build.sbt (hunk 4 above)
Comment (outkaj) [reply build.sbt]: done!

**Pull request merged**